### ISSUE TITLE
Unable to build dev after cleaning test databases.

### ISSUE THREAD

**Issue by anpete** [aspnet/EntityFramework#7381]
First error:

```
  Starting:    Microsoft.EntityFrameworkCore.CrossStore.FunctionalTests
    Microsoft.EntityFrameworkCore.CrossStore.FunctionalTests.ConfigurationPatternsTest.Can_register_multiple_context_types [FAIL]
      System.IO.FileNotFoundException : Could not find file 'C:\dev\github\aspnet\EntityFramework\test\Microsoft.EntityFrameworkCore.CrossStore.FunctionalTests\bin\Debug\netcoreapp1.1\Northwind.sql'.
      Stack Trace:
           at System.IO.FileStream.OpenHandle(FileMode mode, FileShare share, FileOptions options)
           at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 buffe
```

**Comment by rowanmiller** [comment]
We should look at re-enabling the drop all database step on EFCI once this (and other test work) is done.

**Comment by AndriySvyryd** [comment]
Fixed in 87ed1f2a455c5a11977dbb3ea3500141f60c3af7
